source/manage/telemetry.rst

```diff
@@ -195,6 +195,29 @@ The following list details the types of Incident Collaboration metadata we colle
   
 Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. To opt out, disable the feature in **System Console > Environment > Logging**.
 
+Apps Framework Telemetry
```

Comment:
@justinegeffen @larkox There are a few small formatting issues: https://docs.mattermost.com/manage/telemetry.html#apps-framework-telemetry.

(I'm not sure how to fix the backticks not working for bolded letters).